test/Jobs.js

```diff
@@ -4158,4 +4156,192 @@ describe("1100: Jobs: Test New Job Model", () => {
         res.body.should.be.an("array").that.deep.contains({ all: [{ totalSets: 2 }] });
       });
   });
+
+  describe("3120: Validate Job Action", () => {
```

Comment:
This file is massive. I think it would be good to either split it into several files or at least group the tests with additional `describe` blocks.

Grouping them is nice for testing since you can just add `describe.only(` to run tests from a single section.